bin/deploy.sh

```diff
@@ -183,6 +184,29 @@ isalive_check()
     echo "=> AM is alive"
 }
 
+isalive_check_idm()
+{
+    PROTO="https"
+    if [[ -f ${CFGDIR}/openidm.yaml || -f ${CFGDIR}/common.yaml ]]; then
+        if $(grep -sq -e '^tlsStrategy:\s*\bhttp\b' ${CFGDIR}/common.yaml ${CFGDIR}/openidm.yaml); then
+            PROTO="http"
```

Comment:
I was under the impression that you can no longer use tlsStrategy for http - or did we leave it in just for the smoke tests?

Currently it's an undocumented option.

Reply:
Right! My bad, I've just copied whatever we had there before. I will remove this and we go with hardcoded https

Reply:
Removed this for both am & idm